examples/applications/plot_stock_market.py

```diff
@@ -146,7 +149,7 @@
 
 symbols, names = np.array(list(symbol_dict.items())).T
 
-quotes = [finance.quotes_historical_yahoo(symbol, d1, d2, asobject=True)
+quotes = [quotes_historical_yahoo(symbol, d1, d2, asobject=True)
```

Comment:
You will have to do `from matplotlib.finance import quotes_historical_yahoo` at the top, for this to work...